traits/trait_types.py

```diff
@@ -1467,14 +1482,14 @@ def __init__(
         -------------
         *value* or ''
         """
-        if not exists:
-            # Define the C-level fast validator to use:
-            self.fast_validate = (ValidateTrait.coerce, str)
 
         super(File, self).__init__(
             value, filter, auto_set, entries, exists, **metadata
         )
 
+    def validate(self, object, name, value):
```

Comment:
Can this method simply be removed? It looks to me as though it should be fine to simply allow `File` to inherit the `validate` implementation of `BaseFile`.